src/components/networks/utils.js

```diff
@@ -101,28 +86,36 @@ export function validateNetmask(prefixOrNetmask) {
  * @returns {string}
  */
 export function netmaskConvert(prefixOrNetmask) {
-    const parts = prefixOrNetmask.split('.');
+    // single number → netmask
+    if (/^[0-9]+$/.test(prefixOrNetmask)) {
+        try {
+            return ipaddr.IPv4.subnetMaskFromPrefixLength(parseInt(prefixOrNetmask)).toString();
+        } catch (_ex) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1455-20240222-112249-96a33f93-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)